pkgs/applications/science/robotics/dronin/default.nix

```diff
@@ -0,0 +1,120 @@
+{ stdenv, fetchgit, which, python3, gcc-arm-embedded, git, zip, unzip
+, file, dpkg , fakeroot, zlib, libudev, qt59, overrideCC, gcc_multi
+, breakpad, qbs, lib, qtcreator
+}:
+
+let
+  # The simulator is a 32 bit executable
+  stdenv_multi = overrideCC stdenv gcc_multi;
+
+  qt = with qt59;
+       env "qt-dronin-${qtbase.version}" [
+         qtimageformats
+         qttools # for lrelease
+         qtcharts
+         qtdeclarative
+         qtmultimedia
+         qtxmlpatterns
+         qtserialport
+         qtsvg
+       ];
+
+in
+
+stdenv_multi.mkDerivation rec {
+  name = "dRonin-${version}";
```

Comment:
[Don’t use uppercase letters in the name attribute — e.g., "mplayer-1.0rc2" instead of "MPlayer-1.0rc2".](https://nixos.org/nixpkgs/manual/#sec-package-naming).